src/graphql/analytics-overview.graphql

```diff
@@ -0,0 +1,20 @@
+fragment Engagement on Engagement {
+  createdat
+  type
+  count
+  android
+  ios
+  linux
+  macos
+  windows
+  other
+}
+
+query analyticOverview($id: String!) {
+  campaign(id: $id){
+    ...Campaign
+    engagements {
+      ...Engagement
+    }
+  }
+}
```

Comment:
Out of scope for this change, but it occurs to me we might consider discussing pulling these `.graphql` and `.generated.tsx` files into a separate repo that we can import as a package here and `ads-admin-dashboard`.